lib/common/network/api/characters_api.dart

```diff
@@ -10,18 +10,48 @@ part 'characters_api.g.dart';
 abstract class CharactersApi {
   @factoryMethod
   factory CharactersApi(Dio dio) =>
-      FakeApi.characters.isEnabled
-        ? FakeCharactersApi()
-        : _CharactersApi(dio);
+      FakeApi.characters.isEnabled ? FakeCharactersApi() : _CharactersApi(dio);
 
   @GET('')
   Future<Characters> getCharacters();
+
+  @POST('')
+  Future<Characters> addCharacter(
+    @Body() Character character,
+  );
 }
 
 class FakeCharactersApi implements CharactersApi {
+  // variable in FakeCharactersApi for ex fakeCharacters and work with it
+  List<Character> fakeCharacters = [];
+
+  @override
+  Future<Characters> getCharacters() async {
+    if (fakeCharacters.isEmpty) {
```

Comment:
remove it and leave only delay and variable return
regarding prints, I'll show you how to create logs later
```suggestion
     await Future.delayed(Duration(seconds: 2));
     return fakeCharacters;
```

Reply:
Done!)